.pre-commit-config.yaml

```diff
@@ -24,16 +24,6 @@ repos:
   rev: 23.3.0
   hooks:
     - id: black
-- repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Look like an unrelated change to that PR?

Reply:
After all the changes in this PR result was that "There is nothing to check for mypy", thus dropped.